src/components/admin-dashboard/ExportAllLoader.js

```diff
@@ -0,0 +1,42 @@
+import { useMemo } from 'react'
+import CircularProgressBar from './CircularProgressBar'
+import { Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+
+const useStyles = makeStyles(() => ({
+  exportLoader: {
+    position: 'fixed',
+    top: 0,
+    right: '-5%',
+    zIndex: 1000,
+    backdropFilter: 'blur(3px)',
+    // backgroundColor: 'rgba(0,0,0,0.3)',
```

Comment:
If you won't be making any further changes to this file any time soon kindly remove the commented out code @Godstrump